src/analysis/normed_space/normed_group_quotient.lean

```diff
@@ -0,0 +1,489 @@
+/-
+Copyright (c) 2021 Patrick Massot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Massot, Riccardo Brasca
+-/
+import analysis.normed_space.normed_group_hom
+
+/-!
+# Quotients of seminormed groups
+
+For any `semi_normed_group M` and any `S : add_subgroup M`, we provide a `semi_normed_group`
+structure on `quotient_add_group.quotient S` (abreviated `quotient S` in the following).
+If `S` is closed, we provide `normed_group (quotient S)` (regardless of whether `M` itself is
+separated). The main properties of these structures is the underlying topology is the quotient
+topology and the projection is a normed group homomorphism which is norm non-increasing
```

Comment:
I think there's some sort of typo (perhaps an accidentally omitted word?) in this sentence.

Reply:
Thanks. The "is" was meant to be a "are". Let me know if this sentence is still unclear. And thanks for you review!

Reply:
Ah I see. I couldn't figure it out before but it reads fine now.